resources/main.css

```diff
@@ -1,3 +1,13 @@
+a {
+    color: white;
+    font-family: Roboto,sans-serif;
+}
+
+button {
```

Comment:
Do we want a `options.css` file? I then prefer to use identifiers, but we're using `main.css` anyway right now for `options.html`.